app/assets/javascript/entrypoints/career_site.js

```diff
@@ -0,0 +1 @@
+import "../src/shared/bootstrap_activators";
```

Comment:
maybe we move into separate file our logic to turn on bootstrap validation?
smth like `src/career_site/bootstrap_validation`

Reply:
please left comment here too

Reply:
- As far as I understand, we plan to use this in the internal interface as well.
- In general, this applies to bootstrap.
- In the hub, we have the same organization of these files.
- I'm not sure that many small files that will be used in all interfaces will be convenient.

That's why it seems premature to me.